reference/OpenAPI.json

```diff
@@ -27451,11 +27533,14 @@
           },
           "campaign_id": {
             "type": "string",
-            "description": "Unique campaign ID, assigned by Voucherify.",
+            "nullable": true,
+            "description": "Unique campaign identifier, assigned by Voucherify.",
             "example": "camp_weer1c3p5ZgktqfW56RfoNaG"
           },
           "campaign_type": {
             "type": "string",
+            "nullable": true,
```

Comment:
kiedy dostajemy null-a?

Reply:
daje DNM do czasu odp

Reply:
`campaign_id` i `campaign_type` dostają `null`, gdy przypisany "redeemable" jest to kod standalone.